python/ray/train/v2/api/data_parallel_trainer.py

```diff
@@ -163,6 +166,10 @@ def __init__(
         if metadata != _UNSUPPORTED:
             raise NotImplementedError(error_msg.format("metadata"))
 
+        self._run_controller_as_actor = env_bool(
+            RUN_CONTROLLER_AS_ACTOR_ENV_VAR, DEFAULT_RUN_CONTROLLER_AS_ACTOR
+        )
```

Comment:
nit: Evaluate this in `fit` and pass the result in as an arg to both `_initialize_controller` and `_run_controller`? 

We can also consider moving  `_initialize_controller` and `_run_controller` to be global functions in `ray.train.v2._internal.execution.controller.py` if they're not specific to `DataParallelTrainer`

Reply:
makes sense!